include/spdlog/details/os.h

```diff
@@ -136,7 +137,7 @@ inline void prevent_child_fd(FILE *f)
 {
 
 #ifdef _WIN32
-#if !defined(__cplusplus_winrt) && !defined(WINDOWS_UWP) // SetHandleInformation() is missing on UWP
+#if !defined(__cplusplus_winrt) && !(defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_APP)) // SetHandleInformation() is missing on UWP
```

Comment:
this means 'uwp'?

Reply:
Yes, as revealed here: https://social.msdn.microsoft.com/Forums/en-US/51f29cfe-d07f-4898-9962-945ed7b66d7f/predefined-macro-to-identify-uwp